core/orchestrator_core.go

```diff
@@ -4418,6 +5280,15 @@ func (o *TridentOrchestrator) ListVolumePublicationsForNode(
 	return
 }
 
+// listVolumePublicationsForNode returns a list of VolumePublications for a given node
+// Parameters:
+//   nodeName: name of the node to get the list of VolumePublications for
+// Returns:
+//   []*utils.VolumePublication: list of VolumePublications for the given node
```

Comment:
Seems to have gotten confused and listed the type of the return, rather than the name.

Reply:
Whichever, as long as it's consistent